tests/e2e/vLLM/recipes/INT8/recipe_w8a8_dynamic_asym.yaml

```diff
@@ -0,0 +1,17 @@
+quant_stage:
+  quant_modifiers:
+    SmoothQuantModifier:
+      smoothing_strength: 0.8
+      mappings:
+      - - ['re:.*q_proj', 're:.*k_proj', 're:.*v_proj']
+        - re:.*input_layernorm
+      - - ['re:.*gate_proj', 're:.*up_proj']
+        - re:.*post_attention_layernorm
+    GPTQModifier:
+      sequential_update: false
+      ignore: [lm_head]
+      config_groups:
+        group_0:
+          weights: {num_bits: 8, type: int, symmetric: false, strategy: channel}
```

Comment:
weights should be kept symmetric. we'd only want inputs to be asymmetric to leverage vLLM support.